### PR TITLE
fix floating point precision error to calculate width and height for resizing

### PR DESCRIPTION
**Pull request by mwbyeon**
Round the new width and height by multiplying the scale factor to resolve the floating point precision error.

* resolved: #916

### CODE REVIEW

File: dali/pipeline/operators/fused/resize_crop_mirror.h [hunk 1]
```diff
@@ -72,26 +73,26 @@ class ResizeCropMirrorAttr : protected CropAttr {
       // resize_shorter set
       const int shorter_side_size = spec.GetArgument<float>("resize_shorter", ws, index);
       if (meta.H < meta.W) {
-        const float scale = shorter_side_size/static_cast<float>(meta.H);
+        const float scale = shorter_side_size / static_cast<float>(meta.H);
         meta.rsz_h = shorter_side_size;
-        meta.rsz_w = scale * meta.W;
+        meta.rsz_w = static_cast<int>(round(scale * meta.W));
```
Comment (mzient) [inline dali/pipeline/operators/fused/resize_crop_mirror.h]:
```suggestion
        meta.rsz_w = static_cast<int>(std::round(scale * meta.W));
```
`round` will use `::round` from C which takes `double`.

Comment (mwbyeon) [reply dali/pipeline/operators/fused/resize_crop_mirror.h]: @mzient Thank you for your review. I fixed it.

**Comment by klecki** [comment]
!build

**Comment by JanuszL** [comment]
!build

**Pull request merged**